### ISSUE TITLE
"league.free_agents" key error. Football

### ISSUE THREAD

**Issue by SiBensberg** [cwendt94/espn-api#105]
I am trying to get a list of free agents with the following code snippet:
```
league = League(league_id=0101010, year=2020)
freeAgents = league.free_agents(size=5)
```
I am getting the following error:
```
Traceback (most recent call last):
  File "/home/simon/espn_api_workspace/main.py", line 11, in <module>
    freeAgents = league.free_agents(size=5)
  File "/home/simon/anaconda3/envs/fantasy_espn/lib/python3.8/site-packages/espn_api/football/league.py", line 261, in free_agents
    return [BoxPlayer(player, pro_schedule, positional_rankings, week) for player in players]
  File "/home/simon/anaconda3/envs/fantasy_espn/lib/python3.8/site-packages/espn_api/football/league.py", line 261, in <listcomp>
    return [BoxPlayer(player, pro_schedule, positional_rankings, week) for player in players]
  File "/home/simon/anaconda3/envs/fantasy_espn/lib/python3.8/site-packages/espn_api/football/box_player.py", line 38, in __init__
    self.projected_breakdown = {PLAYER_STATS_MAP.get(int(k), k):v for (k,v) in stats['appliedStats'].items()}
KeyError: 'appliedStats'
```

In the debugger there is no key 'applied stats' in ```box_player```. 

I changed the `'appliedStats'` key in `espn_api/football/box_player.py` to just `'stats` and I am getting a somewhat reasonable breakdown list. (line 33 + line 36)
Some keys don't have names and I don't know their meaning.

![image](https://user-images.githubusercontent.com/55738346/92097546-7c28f800-edd8-11ea-8029-ac51cffdfa49.png)

Is this a bug or am I doing something wrong?

**Comment by cwendt94** [comment]
That is a bug! I'm not 100% sure the different between the `stats` key and `appliedStats` because I am seeing `appliedStats` for box score. It might be projected stats before the season starts. I will push a fix for this!

**Comment by cwendt94** [comment]
Fixed in v0.4.2
